components/LeagueCard/LeagueCard.test.tsx

```diff
@@ -85,4 +85,21 @@ describe('LeagueCard with custom league logo', () => {
     expect(leagueCardTitle).toBeInTheDocument();
     expect(leagueCardTitle).toHaveTextContent('League 1');
   });
+
+  it('should apply hover effect on mouse hover', () => {
+    render(
+      <LeagueCard
+        href="/leagues"
+        isEliminated={true}
+        survivors={11}
+        title="League 1"
+        totalPlayers={12}
+      />,
+    );
+    const leagueCard = screen.getByTestId('LeagueCard');
+    fireEvent.mouseOver(leagueCard);
+    expect(leagueCard).toHaveClass(
+      ' LeagueCard flex max-h-32 place-items-center gap-6 rounded-lg border bg-card p-4 text-card-foreground shadow-sm dark:border-zinc-800 hover:bg-zinc-800 transition',
```

Comment:
The only class you need to check for is `hover:bg-zinc-800`. Do not check for the other clasnames.

Reply:
Yep, I did think of this but there were no Jest matchers that I could use to help silo out only `hover:bg-zinc-800`. Do you by chance know any Jest matchers I could use to check this?

Reply:
You can do a `toHaveClass` function. Look at how Button component test does it..